axelor-base/src/main/resources/views/PriceList.xml

```diff
@@ -37,7 +37,7 @@
 		<panel name="priceListLinePanel" title="Price list lines" >
 			<field name="generalDiscount"/>
 			<label name="orLabel" title="Or" colSpan="12"/>
-			<panel-related name="historizedPriceListPanel" field="priceListLineList" grid-view="price-list-line-grid" form-view="price-list-line-form" colSpan="12"/>
+			<panel-related name="PriceLineListPanel" field="priceListLineList" grid-view="price-list-line-grid" form-view="price-list-line-form" colSpan="12"/>
```

Comment:
To be consistent with other panel titles, please start with a lowercase ( `PriceLineListPanel` => `priceLineListPanel`)